### PR TITLE
Deleted unnecessary code

### PR DESCRIPTION
**Pull request by emrekasg**

### CODE REVIEW

File: apiserver/firestoreutil/testutil.go [hunk 1]
```diff
@@ -15,10 +15,9 @@
 package firestoreutil
 
 import (
+	"cloud.google.com/go/firestore"
 	"context"
 	"testing"
-
-	firestore "cloud.google.com/go/firestore"
```
Comment (ahmetb) [inline apiserver/firestoreutil/testutil.go:21]: don't change these. external imports are typically in separate blocks. see other files.

File: apiserver/main.go [hunk 2]
```diff
@@ -25,11 +25,11 @@ import (
...
 
 	"github.com/google/uuid"
-	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
-	grpc_auth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
-	grpc_zap "github.com/grpc-ecosystem/go-grpc-middleware/logging/zap"
+	grpcmiddleware "github.com/grpc-ecosystem/go-grpc-middleware"
+	grpcauth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
+	grpczap "github.com/grpc-ecosystem/go-grpc-middleware/logging/zap"
 	"github.com/grpc-ecosystem/go-grpc-middleware/logging/zap/ctxzap"
-	grpc_ctxtags "github.com/grpc-ecosystem/go-grpc-middleware/tags"
```
Comment (ahmetb) [inline apiserver/main.go:32]: don't change these.

File: apiserver/main.go [hunk 3]
```diff
@@ -136,21 +136,21 @@ func main() {
 }
 
 func prepServer(log *zap.Logger, au auth.Authenticator, rl ratelimiter2.RateLimiter, udb *userdb.UserDB, as *accountService, ts *tickerService, pt *tradingService) *grpc.Server {
-	unaryInterceptors := grpc_middleware.WithUnaryServerChain(
+	unaryInterceptors := grpcmiddleware.WithUnaryServerChain(
 		otelgrpc.UnaryServerInterceptor(),
-		grpc_ctxtags.UnaryServerInterceptor(grpc_ctxtags.WithFieldExtractor(grpc_ctxtags.CodeGenRequestFieldExtractor)),
-		grpc_zap.UnaryServerInterceptor(log),
+		grpcctxtags.UnaryServerInterceptor(grpcctxtags.WithFieldExtractor(grpcctxtags.CodeGenRequestFieldExtractor)),
```
Comment (ahmetb) [inline apiserver/main.go:141]: don't change these for no reason.

**Comment by emrekasg** [comment]
OK Just sent a commit.

**Pull request closed without merging**